AIDevGallery/Controls/ModelPicker/ModelOrApiPicker.xaml.cs

```diff
@@ -216,13 +217,10 @@ private async Task LoadModels(List<ModelType> types)
 
         foreach (var def in pickers)
         {
-            if (def.Id == "ollama")
+            if ((def.Id == "ollama" && !await OllamaModelProvider.Instance.IsAvailable())
```

Comment:
As we spoke offline, only feedback is abstracting this into the interface somehow.